addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -50,6 +50,7 @@
                     to
                     <span t-out="highest_price" t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.currency_id}"/>
                 </t>
+                <i>(Tax Excluded)</i>
```

Comment:
the translation should be exported in the `addons/website_event_sale/i18n/website_event_sale.pot'